packages/firestore/src/api/database.ts

```diff
@@ -138,9 +149,10 @@ import {
   WhereFilterOp as PublicWhereFilterOp,
   WriteBatch as PublicWriteBatch
 } from '@firebase/firestore-types';
-import { newUserDataReader } from '../../lite/src/api/reference';
+
 import { makeDatabaseInfo } from '../../lite/src/api/database';
 import { DEFAULT_HOST } from '../../lite/src/api/components';
+import * as exp from '../../exp/index';
```

Comment:
I thought `*` imports were to be avoided?

Reply:
Yeah, I should have caught this. Removed this import and aliased DocumentReference to ExpDocumentReference.